pyproject.toml

```diff
@@ -65,6 +66,7 @@ dev = [
 	"pygments",
 	"astor",
 	"pyproj",	# for running guideways modules
+	"pegen",
 ]
 test = [	# minimum dependencies for running tests (used for tox virtualenvs)
```

Comment:
We don't necessarily need to do this now, but eventually `pegen` will probably have to be included in the `test` extra as well since the test suite should check that the grammar compiles and then use the resulting parser.

Reply:
> the test suite should check that the grammar compiles and then use the resulting parser

Forgot about this. Thanks for letting me know.

Also added this on Todoist. I will work on this when I will add tests for the parser generation and use the resulting parser for the rest of the test suite.